docker-compose.yml

```diff
@@ -11,8 +11,10 @@ services:
             - python
             - python-sync
             - redis
+            - cherami
         environment:
-            - WAIT_FOR=go,node,python,python-sync
+            - WAIT_FOR=go,node,python,python-sync,cherami
```

Comment:
Here's some docs:

- https://docs.docker.com/compose/compose-file/#/dependson
- https://docs.docker.com/compose/compose-file/#/healthcheck
- https://docs.docker.com/engine/reference/builder/#/healthcheck

Note that you can define the health check either in Dockerfile or in docker-compose.yml, depending on your usecase. Defining the check in docker-compose.yml will override whatever is in the Dockerfile.